rs/sns/integration_tests/src/initialization_flow.rs

```diff
@@ -303,6 +304,32 @@ impl SnsInitializationFlowTestSetup {
     }
 }
 
+// TODO: Reuse this function from ic_sns_test_utils::state_test_helpers once that crate is ported
+// TODO: to use `ic_sns_governance_api` (rather than `ic_sns_governance`).
+fn sns_governance_list_neurons(
```

Comment:
The idea is to reuse as much functionality as possible from ic_nervous_system_agent. This is here temporarily.